CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+CMAKE_MINIMUM_REQUIRED(VERSION 3.4.3)
```

Comment:
Please add comment here outlining purpose of this file

Reply:
Done.